src/NServiceBus.Testing.Fakes/TestableAuditContext.cs

```diff
@@ -17,12 +18,12 @@ public class TestableAuditContext : TestableBehaviorContext, IAuditContext
         /// <summary>
         /// Address of the audit queue.
         /// </summary>
-        public string AuditAddress { get; set; }
+        public string AuditAddress { get; set; } = "audit queue address";
```

Comment:
Would "audit" be a better default value for this? In the normal pipeline execution, would this already be a value that has been transformed by the transport? 

I suppose for testing purposes the actual value doesn't matter.